App/Sources/DesignSystem/View/MainMissionScrollView.swift

```diff
@@ -0,0 +1,47 @@
+import Service
+import SwiftUI
+
+struct MainMissionScrollView: View {
+    let missionDataList: [MissionListEntity]
+    
+    var body: some View {
+        VStack {
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack(spacing: 16) {
+                    ForEach(missionDataList.indices, id: \.self) { mission in
+                        if let missionList = missionDataList[safe: mission] {
+                            VStack {
+                                Text(missionList.user.userName)
+                                    .skFont(.pm14)
+                                    .multilineTextAlignment(.center)
+                                    .padding(.top, 16)
+                                
+                                Text(missionList.missionTitle)
+                                    .skFont(.pr12)
+                                    .multilineTextAlignment(.center)
+                                    .lineLimit(3)
+                                    .padding(.top, 16)
+                                    .padding(.horizontal, 16)
+                                
+                                Spacer()
+                                
+                                Text(missionList.missionPoint.description)
+                                    .skFont(.pm16)
+                                    .multilineTextAlignment(.center)
```

Comment:
```suggestion
                                    .multilineTextAlignment(.center)
```
이거 VStack에 alignment center주는 걸로 해결 안되던가용? 같은 코드가 Text별로 적용되어있어서 궁금해요;

Reply:
어이쿠 너무 생각없이 코드를 집어넣었던 거 같네요;; 각 text의 alignment 지정해준거 빼주고 vstack에 alignment를 지정해주도록 하겠습니다~